[BUG] Error: Invariant: renderHTML should not be called in minimal mode

### Describe the bug

I'm trying to make a post to the following NextJS API route and get this error back after the wrangler / opennext build. Not sure what's causing it

```
import { NextRequest, NextResponse } from 'next/server';
import { isValidHandle } from '@atproto/syntax'

import { client as atprotoOauthClient } from '@/lib/atproto/oauth/client'

type Session = { did: string }

export async function POST(req: NextRequest) {
  const data = await req.json()

  // Validate
  const handle = data.handle
  if (typeof handle !== 'string' || !isValidHandle(handle)) {
    return NextResponse.json({error: "invalid handle"})
  }

  // Initiate the OAuth flow
  try {
    const url = await atprotoOauthClient.authorize(handle, {
      scope: 'atproto transition:generic',
    })
    return NextResponse.json({ redirect: url.toString() })
  } catch (err) {
    return NextResponse.json({error: "oauth authorize failed", details: err, message: err.message})
  }
}
```

### Steps to reproduce

Not sure yet, can get CF trace details

I can see about producing a minimal reproducer in the next couple of days

### Expected behavior

I can post to oauth endpoint and not get errors

### @opennextjs/cloudflare version

0.5.3

### Wrangler version

3.109.2

### next info output

```shell
Operating System:
  Platform: darwin
  Arch: arm64
  Version: Darwin Kernel Version 24.3.0: Thu Jan  2 20:23:36 PST 2025; root:xnu-11215.81.4~3/RELEASE_ARM64_T8112
  Available memory (MB): 24576
  Available CPU cores: 8
Binaries:
  Node: 21.7.1
  npm: 10.5.0
  Yarn: 1.22.21
  pnpm: 8.14.0
Relevant Packages:
  next: 15.1.7 // Latest available version is detected (15.1.7).
  eslint-config-next: 15.1.0
  react: 19.0.0
  react-dom: 19.0.0
  typescript: 5.7.3
Next.js Config:
  output: standalone
```


### Additional context

running site is at https://verdverm.verdverm.workers.dev

@vicb still seems like the real error is being hidden. I'm now seeing

`✘ [ERROR] ⨯ TypeError: Cannot read properties of undefined (reading 'definition')`

Though the third line on the minified stack trace looks to indicate it wants to return a string about an unsupported api usage

I updated wrangler and opennext to the latest in my repro: https://github.com/verdverm/cf-repro

--- a/.changeset/cyan-ears-know.md
+++ b/.changeset/cyan-ears-know.md
@@ -1,0 +1,5 @@
+---
+"@opennextjs/cloudflare": patch
+---
+
+Define `process.version` and `process.versions.node`

--- a/packages/cloudflare/src/cli/templates/worker.ts
+++ b/packages/cloudflare/src/cli/templates/worker.ts
@@ -1,4 +1,5 @@
 import { AsyncLocalStorage } from "node:async_hooks";
+import process from "node:process";
 
 import type { CloudflareContext } from "../../api";
 // @ts-expect-error: resolved by wrangler build
@@ -67,6 +68,13 @@ function populateProcessEnv(url: URL, env: CloudflareEnv) {
   if (processEnvPopulated) {
     return;
   }
+
+  // Some packages rely on `process.version` and `process.versions.node` (i.e. Jose@4)
+  // TODO: Remove when https://github.com/unjs/unenv/pull/493 is merged
+  Object.assign(process, { version: process.version || "v22.14.0" });
+  // @ts-expect-error Node type does not match workerd
+  Object.assign(process.versions, { node: "22.14.0", ...process.versions });
+
   processEnvPopulated = true;
 
   for (const [key, value] of Object.entries(env)) {

